assets/js/googlesitekit/datastore/site/urls.test.js

```diff
@@ -77,6 +77,157 @@ describe( 'core/site urls', () => {
 			} );
 		} );
 
+		describe( 'getDocumentationLinkURL', () => {
+			it( 'should throw an error if the slug is omitted', () => {
+				try {
+					registry.select( CORE_SITE ).getDocumentationLinkURL();
+				} catch ( error ) {
+					expect( error.message ).toBe( 'A slug is required.' );
+				}
+			} );
```

Comment:
This can be simplified to follow our other tests:
```suggestion
			it( 'should throw an error if the slug is omitted', () => {
			        expect( () => {
					registry.select( CORE_SITE ).getDocumentationLinkURL();
				} ).toThrow( 'A slug is required.' );
			} );
```